src/MatrixSize.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+
+public class MatrixSize {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int i = 0;
+        String temp = "";
+        while (scanner.hasNextLine()) {
+            String arr = scanner.nextLine();
+            i++;
+            temp = arr.charAt(arr.length() - 1) + "";
+        }
+        int columns = Integer.parseInt(temp);
+        columns = columns/i;
```

Comment:
I don't think this works. Try this input:
111
111

Reply:
I know, later I will fix it